src/updater.cpp

```diff
@@ -77,7 +78,7 @@ void Updater::checkUpdatesOnStart()
     mDailyCheck->setInterval(12h);
     connect(mDailyCheck.get(), &QTimer::timeout, this, [this] {
           auto updates = feed->getUpdates(dblsqd::Release::getCurrentRelease());
-          qWarning() << "Daily check for updates:" << updates.size() << "update(s) available";
+          qWarning() << "Bi-daily check for updates:" << updates.size() << "update(s) available";
```

Comment:
This seems unrelated to the original issue, and should be fast-tracked if applicable.